ehrapy/data/_dataloader.py

```diff
@@ -61,3 +61,7 @@ def download(
     if is_archived:
         output_path = output_path or tempfile.gettempdir()
         shutil.unpack_archive(download_to_path, output_path)
+
+    logg.info(
```

Comment:
Don't think that we need logging messages for this or do we? @Imipenem opinion?

Reply:
Hm, could actually be helpful (especially since this code part was a bit tricky initially), so I'd say it's fine.